system/lib/libc/musl/src/stat/futimens.c

```diff
@@ -2,5 +2,10 @@
 
 int futimens(int fd, const struct timespec times[2])
 {
+#if __EMSCRIPTEN__
+	#define AT_EMPTY_PATH 0x1000
+	return utimensat(fd, 0, times, AT_EMPTY_PATH);
+#else
 	return utimensat(fd, 0, times, 0);
```

Comment:
Actually looking at the man page for utimensat.. it doesn't mention support for AT_EMPTY_PATH (unlike for example the man page for fstatat).futimens

Also, AT_EMPTY_PATH I believe talks about the empty string filename, not the null filename..  I wonder if we should revert this part of the patch and handle NULL path in __sys_utimensat as linux seems to?  The easiest way to do that might be to just always pass True for allowEmpty?

Reply:
Removed the flags code and just passing true now.